gormsupport/db_test_suite.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/almighty/almighty-core/configuration"
+	c "github.com/almighty/almighty-core/configuration"
```

Comment:
In order to not confuse the reader, please try to use the same import names in different files. I remember you called it `config` above, right?